plugins/modules/sysrc.py

```diff
@@ -122,14 +122,19 @@ def has_unknown_variable(self, out, err):
         return err.find("unknown variable") > 0 or out.find("unknown variable") > 0
 
     def exists(self):
-        # sysrc doesn't really use exit codes
-        (rc, out, err) = self.run_sysrc(self.name)
+        """
+        Test the name is in the file. If a value is defined test name=value
+        is in the file. These tests are necessary because sysrc doesn't use exit
+        codes. Instead, let sysrc read the file's content and create a
+        dictionary comprising the configuration. Use this dictionary to preform
+        the tests.
```

Comment:
Just wordsmithing a bit:
```suggestion
        Tests whether the name is in the file. 
        If parameter value is defined, then tests whether name=value is in the file. 
        These tests are necessary because sysrc doesn't use exit codes. 
        Instead, let sysrc read the file's content and create a dictionary comprising the configuration. 
        Use this dictionary to preform the tests.
```